app/api/src/functions/identity-signup.js

```diff
@@ -2,6 +2,7 @@ import { createUserInsecure } from 'src/services/users/users.js'
 import { db } from 'src/lib/db'
 import { sentryWrapper } from 'src/lib/sentry'
 import { enforceAlphaNumeric, generateUniqueString } from 'src/services/helpers'
+import 'graphql-tag'
```

Comment:
and explicitly requiring this dependency for my signup function

Reply:
From: https://community.redwoodjs.com/t/redwood-v0-32/2118/7

Ah - you said the magic word trifecta of Netlify + Functions + Dependency and I wonder if you encountered this: https://www.netlify.com/blog/2021/04/02/modern-faster-netlify-functions/

> With the latest release of [our function bundler](https://github.com/netlify/zip-it-and-ship-it), we’re starting to use [esbuild](https://esbuild.github.io/) under the hood to handle some parts of this. It also includes an additional step of *inlining* , where your function code and its dependencies are physically merged into a single file.

> The new bundler will be enabled for all projects during the week of May 17, but you can choose to opt-in right now to test the new functionality in public beta and take advantage of the performance improvements immediately.

Perhaps the way Netlify is bundling now wasn't finding that dependency?

And v.0.32 was released on May 14 so ... "week of May 17" does coincide.

Thus `make graphql-tag and explicit dependency` fixing the issue fits.Thus `make graphql-tag and explicit dependency` fixing the issue fits.

> You can define a list of modules that should be copied to the generated function artifact with their source and references untouched, skipping the inlining and tree-shaking stages. This is useful for handling dependencies that can’t be inlined, such as modules with native addons.

> This is done with the `external_node_modules` property, which you can apply to all functions, or filter some them by name using a [wildcard pattern](https://en.wikipedia.org/wiki/Wildcard_character).

```terminal
# In your netlify.toml
# All functions
[functions]
  external_node_modules = ["module-one", "module-two"]

# Functions with a name starting with "my-function-*"
[functions."my-function-*"]
  external_node_modules = ["module-three", "module-four"]

# A function named "my-function-1"
[functions.my-function-1]
  external_node_modules = ["module-five", "module-six"]
```